dockstore-webservice/src/main/java/io/dockstore/webservice/core/Notebook.java

```diff
@@ -17,12 +17,17 @@
 
 import io.dockstore.common.EntryType;
 import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.persistence.Entity;
 import jakarta.persistence.NamedQueries;
 import jakarta.persistence.NamedQuery;
+import jakarta.persistence.OneToOne;
 import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+
+@ApiModel(value = "Notebook", description = Notebook.NOTEBOOK_DESCRIPTION, parent = Workflow.class)
```

Comment:
This line doesn't effectively change anything, does it, since we're not generating Swagger any more? Just making sure I understand.

Reply:
You're right. I think I was just cleaning up the description for DRY.